pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -35,20 +37,20 @@ const (
 )
 
 var (
-	healthCheckInterval = 300 * time.Second
+	healthCheckInterval = 3 * time.Minute
```

Comment:
Shouldn't this be 5 minutes instead?

Reply:
I'm going to revert it, was just testing with a shorter time.